packages/react-components/lib/tasks/task-info.tsx

```diff
@@ -26,108 +26,25 @@ function InfoValue({ children }: React.PropsWithChildren<unknown>) {
 }
 
-// interface CleanTaskInfoProps {
-//   task: TaskState;
-// }
-
-// function CleanTaskInfo({ task }: CleanTaskInfoProps) {
-//   return (
-//     <InfoLine>
-//       <span>Start Waypoint:</span>
-//       <span style={{ float: 'right' }}>{parseTaskDetail(task, task?.category).to}</span>
-//     </InfoLine>
-//   );
-// }
-
-// interface LoopTaskInfoProps {
-//   task: TaskState;
-// }
-
-// function LoopTaskInfo({ task }: LoopTaskInfoProps) {
-//   return (
-//     <>
-//       <InfoLine>
-//         <span>Start Waypoint:</span>
-//         <InfoValue>{parseTaskDetail(task, task?.category).from}</InfoValue>
-//       </InfoLine>
-//       <InfoLine>
-//         <span>Finish Waypoint:</span>
-//         <InfoValue>{parseTaskDetail(task, task?.category).to}</InfoValue>
-//       </InfoLine>
-//       <InfoLine>
-//         <span>Num of Loops:</span>
-//         <InfoValue>{task.phases ? Object.keys(task.phases).length / 2 : null}</InfoValue>
-//       </InfoLine>
-//     </>
-//   );
-// }
-
-// interface DeliveryTaskInfoProps {
-//   task: TaskState;
-// }
-
-// function DeliveryTaskInfoProps({ task }: DeliveryTaskInfoProps) {
-//   // TODO - replace all temp values
-//   return (
-//     <>
-//       <InfoLine>
-//         <span>Pickup Location:</span>
-//         <span style={{ float: 'right' }}>{'temp'}</span>
-//       </InfoLine>
-//       <InfoLine>
-//         <span>Pickup Dispenser:</span>
-//         <span style={{ float: 'right' }}>{'temp'}</span>
-//       </InfoLine>
-//       <InfoLine>
-//         <span>Dropoff Location:</span>
-//         <span style={{ float: 'right' }}>{parseTaskDetail(task, task?.category).from}</span>
-//       </InfoLine>
-//       <InfoLine>
-//         <span>Dropoff Ingestor:</span>
-//         <span style={{ float: 'right' }}>{parseTaskDetail(task, task?.category).to}</span>
-//       </InfoLine>
-//     </>
-//   );
-// }
-
 export interface TaskInfoProps {
   task: TaskState;
+  title?: string;
 }
 
-export function TaskInfo({ task }: TaskInfoProps): JSX.Element {
+export function TaskInfo({ task, title }: TaskInfoProps): JSX.Element {
   const theme = useTheme();
-  // const taskType = task.category;
-  // const detailInfo = (() => {
-  //   switch (taskType) {
-  //     case 'Clean':
-  //       return <CleanTaskInfo task={task} />;
-  //     case 'Loop':
-  //       return <LoopTaskInfo task={task} />;
-  //     case 'Delivery':
-  //       return <DeliveryTaskInfoProps task={task} />;
-  //     default:
-  //       return null;
-  //   }
-  // })();
 
   return (
     <StyledDiv>
       <Typography variant="h6" style={{ textAlign: 'center' }} gutterBottom>
-        {task.booking.id}
+        {title ? title : task.booking.id}
       </Typography>
       <Divider />
       <div style={{ marginBottom: theme.spacing(1) }}></div>
       <InfoLine>
         <span>State:</span>
         <InfoValue>{task.status || 'unknown'}</InfoValue>
       </InfoLine>
```

Comment:
Referring back to figma, I believe we may have shuffled up some of the intended fields, `Start waypoint` and `Finish waypoint` should be below this `Status`/`State` field within the `TaskInfo` components. Migrating the waypoint extraction logic down to this component will probably make `TaskInspector` much cleaner too

Reply:
Perfect!
Nice suggestion!